examples/utils/example-utils/src/migrationInterfaces/migratableModel.ts

```diff
@@ -47,13 +45,11 @@ export interface IMigratableModel
 	extends IVersionedModel,
 		IImportExportModel<unknown, unknown> {
 	/**
-	 * The tool that will be used to facilitate the migration.
-	 */
-	readonly migrationTool: IMigrationTool;
```

Comment:
This is the main win - customers don't have to provide the IMigrationTool anymore when implementing the IMigratableModel interface.